server/driver.go

```diff
@@ -71,7 +71,7 @@ type QueryCtx interface {
 	ExecuteStmt(context.Context, ast.StmtNode) (ResultSet, error)
 
 	// Parse parses a SQL to statement node.
-	Parse(ctx context.Context, sql string) ([]ast.StmtNode, error)
+	Parse(ctx context.Context, sql string) ([]ast.StmtNode, []error, error)
```

Comment:
Need comment on the returned []error

Reply:
After merging master,  I find @eurekaka has fixed this.